core/pom.xml

```diff
@@ -194,6 +194,11 @@
             <artifactId>jna</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
I think we've been trying to remove dependencies so adding this to core in exchange for some Guava classes is probably going to upset someone.